src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/http/IssuanceRequestJsonMapper.kt

```diff
@@ -68,7 +69,20 @@ internal data class BatchCredentialResponseSuccessTO(
     }
 
     companion object {
-        fun from(jwtClaimsSet: JWTClaimsSet): BatchCredentialResponseSuccessTO = TODO()
+        fun from(jwtClaimsSet: JWTClaimsSet): BatchCredentialResponseSuccessTO = with(jwtClaimsSet) {
+            val jsonArray = JSONObjectUtils.getJSONObjectArray(claims, "credential_responses")
+            BatchCredentialResponseSuccessTO(
+                credentialResponses = jsonArray.map {
+                    IssuanceResponseTO(
+                        credential = JSONObjectUtils.getString(it, "credential"),
```

Comment:
Consider renaming `it`, to something like `res`

It is defined too far way from where it is being used, IMO.